src/privileged/notificationBar/api.js

```diff
@@ -49,38 +49,47 @@ class NotificationBarEventEmitter extends EventEmitter {
   emitShow(variationName) {
     const self = this;
     const recentWindow = getMostRecentBrowserWindow();
-    const doc = recentWindow.document;
-
     const browser = recentWindow.gBrowser.selectedBrowser;
     const tabId = tabTracker.getBrowserTabId(browser);
 
     const primaryAction =  {
-      label: "Yes!",
+      disableHighlight: true,
+      label: "Yes (broken)",
       accessKey: "f",
       callback: () => {
-        self.emit("page-broken", {tabId});
+        self.emit("page-broken", tabId);
       },
     };
 
     const secondaryActions =  [
       {
-        label: "Nope",
+        label: "No (works)",
         accessKey: "d",
         callback: () => {
-          self.emit("page-not-broken", {tabId});
+          self.emit("page-not-broken", tabId);
         },
       },
     ];
 
-    const populatePanel = (event) => {
-      // console.log("EVENT HAPPENED", event);
-      // Can edit the contents of the panel here if we want variations per branch
-      // if (event !== "shown") {
-      //   return;
-      // }
-    };
+    // option persistant is described as: "the notification will always
+    //                    persist even across tab and app changes (but not across
+    //                    location changes), until the user accepts or rejects
+    //                    the request. The notification will never be implicitly
+    //                    dismissed."
+    // This is not the case, it does not persist across tab changes
 
-    doc.defaultView.PopupNotifications.show(browser, "fast-block-notification", "Is this page broken?", null, primaryAction, secondaryActions, {eventCallback: populatePanel});
+    // option name is described as: "An optional string formatted to look bold and used in the
+    //                    notifiation description header text. Usually a host name or
+    //                    addon name."
+    // It is bold, but not used in a header, we're working with it anyway.
+    const options = {
+      hideClose: true,
+      persistent: true,
+      autofocus: true,
+      name: "Firefox Survey: ",
+      popupIconURL: "chrome://branding/content/icon64.png",
+    };
+    recentWindow.PopupNotifications.show(browser, "fast-block-notification", "<> Did you reload this page to resolve loading issues?", null, primaryAction, secondaryActions, options);
```

Comment:
I feel like we should open a (backlog) issue to add translation files for this. There are examples from the TRR Study (for using WebExtension JSON locale files) and the Fx Monitor Study (for using .properties files) on how to do this.

Reply:
Yes, but aren't we only testing in EN locales? I don't think we do shield studies outside of EN.